kolibri/plugins/coach/test/helpers.py

```diff
@@ -0,0 +1,55 @@
+from kolibri.core.auth.models import FacilityUser
+
+
+def create_learner(username, password, facility, classroom=None, learner_group=None):
```

Comment:
Maybe in the future we can just assume every test user is using "DUMMY_PASSWORD" so it can be set by default in these kinds of helpers.

Reply:
Same with any other constants, like the facility

Reply:
👍

Reply:
I've just written it down and am going to include it in tests simplifications proposal.